ultravox/evaluation/gpt_eval.py

```diff
@@ -18,9 +19,9 @@
 """
 INSTRUCT_USER_PROMPT = """
 Using the supplied correct answer as an example, evaluate the model's ability to follow the instructions in the question below:
-Question: {question}
-Model answer: {generated_answer}
-Correct answer: {expected_answer}
+Question: {{ question }}
```

Comment:
should we move these out to .jinja files too?

Reply:
Hmm, possible.

Not a strong opinion, but I'd rather have the system and user messages in the same place.
We can do that with Jinja, but that might get complicated again.

Reply:
yeah, makes sense. maybe just split each eval and its prompts out into its own .py for now

Reply:
Done. Separated evals and added a test for conversation eval because the Jinja template was getting messy.